modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -7,200 +7,180 @@
+* An external source code repositories provider (Git)
 
-[NOTE]
-====
-When an OpenShift cluster contains cluster-wide trusted CA certificates added through the link:https://docs.openshift.com/container-platform/4.10/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki[cluster-wide-proxy configuration], {prod-short} Operator detects them and automatically injects them into a `ConfigMap` object. {prod-short} automatically labels the `ConfigMap` object with the `config.openshift.io/inject-trusted-cabundle="true"` label. Based on this annotation, OpenShift automatically injects the cluster-wide trusted CA certificates inside the `ca-bundle.crt` key of the `ConfigMap` object.
-====
+{prod-short} uses labeled config maps in {prod-short} {orch-namespace} as sources for TLS certificates.
+The config maps can have an arbitrary amount of keys with a random amount of certificates each.
 
-[IMPORTANT]
+[NOTE]
 ====
-Some {prod-short} components require a full certificate chain to trust the endpoint.
-If the cluster is configured with an intermediate certificate, add the whole chain, including self-signed root, to {prod-short}.
+When an OpenShift cluster contains cluster-wide trusted CA certificates added through the link:https://docs.openshift.com/container-platform/latest/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki[cluster-wide-proxy configuration],
+{prod-short} Operator detects them and automatically injects them into a config map with the `config.openshift.io/inject-trusted-cabundle="true"` label.
+Based on this annotation, OpenShift automatically injects the cluster-wide trusted CA certificates inside the `ca-bundle.crt` key of the config map.
 ====
 
-== Adding new CA certificates into {prod-short}
-
-Certificate files are typically stored as Base64 files, with extensions such as `.pem`, `.crt`, `.ca-bundle`, and others. All Secrets that hold certificate files should use the Base64-encoded certificate rather than binary-encoded certificate. The following procedure is applicable for already installed and running instances and for instances that are to be installed.
-
 .Prerequisites
+* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster.
+See {orch-cli-link}.
 
-* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
-* Namespace for {prod-short} exists.
-* {prod-short} already uses some reserved file names to automatically inject certificates into the `ConfigMap` object. Avoid using the following reserved file names to save your certificates:
-  ** `ca-bundle.crt`
-  ** `ca.crt`
+* The `{prod-namespace}` {orch-namespace} exists.
 
-.Procedure
+* The complete Certificate Authorities chains to import (root CA and intermediate) in link:https://wiki.openssl.org/index.php/PEM[PEM] format in `ca-cert-__<count>__.pem` files.
```

Comment:
```suggestion
* The complete Certificate Authorities chains (root CA and intermediate) are ready for import in link:https://wiki.openssl.org/index.php/PEM[PEM] format in `ca-cert-__<count>__.pem` files.
```

Reply:
I believe with the suggestion we loose the meaning: we must have this: "the complete Certificate Authorities chains to import", with additional conditions:  "in PEM format", and "in ca-certs-<coun>.pem files".

Reply:
ok, I see. Leave it as is, then.

Reply:
I tried something for clarity.